docs/semgrep-secrets/historical-scanning.md

```diff
@@ -0,0 +1,119 @@
+---
+slug: historical-scanning
+append_help_link: true
+title: Scan your Git history (beta)
+hide_title: true
+description: Detect valid, leaked secrets in previous Git commits through a historical scan.
+tags:
+  - Semgrep Secrets
+  - Team & Enterprise tier
+---
+
+# Scan your Git history (beta)
+
+Detect valid, leaked secrets in previous Git commits through a **historical scan**.
+
+You can perform one-time historical scans or enable historical scanning for full Secrets scans. Detecting valid secrets in your Git history is a step to reducing your repository's attack surface.
+
+You can run historical scans in the CLI and in your Semgrep deployment, which enables you to track and triage these secrets.
+
+:::note Feature maturity
+- This feature is in **public beta**. See [Limitations](#limitations) for more information.
+- All Semgrep Secrets customers can enable this feature.
+- Please leave feedback by either reaching out to your technical account manager (TAM) or through the **<i class="fa-solid fa-bullhorn"></i> Feedback** form in Semgrep Cloud Platform's navigation bar.
+:::
+
+## Run historical scans
+
+You can enable historical scanning for your full scans or run a dedicated CI job for one-time scans. By default, historical scans display **valid, leaked secrets** to ensure a high true positive rate. Historical scans do **not** run on diff scans.
+
+### Prerequisites
+
+- **CLI tool**: Historical scanning requires at least Semgrep **v1.65.0**. See [Update](/update/) for instructions.
+- **Single-tenant Semgrep Cloud Platform (SCP)**: Reach out to your TAM to ensure your instance is up-to-date.
+
+### Enable historical scanning for full Secrets scans
+
+:::tip
+If possible, [test historical scans locally](#run-a-local-test-scan) to create a benchmark of performance and scan times before adding historical scans to your formal security process.
+:::
+
+1. Sign in to Semgrep Cloud Platform.
+1. Click **<i class="fa-solid fa-gear"></i> Settings**.
+1. Under Deployment, click the **<i class="fa-solid fa-toggle-large-on"></i> Historical scanning** toggle.
+![Historical scanning settings toggle](/img/historical-scanning-settings.png#md-width)
+
+Subsequent Semgrep full scans now include historical scanning.
+
+### Run a one-off historical scan
+
+To run a one-off or on-demand historical scan, you can create a specific CI job and then manually start the job as needed.
+
+The general steps are:
+
+1. Copy your current full scan CI job configuration file, or use [a template](/semgrep-ci/sample-ci-configs/).
+1. Look for the `semgrep ci` command.
```

Comment:
same question

Reply:
In markdown, we write:
```
1.
1.
1.
```
Because they are rendered as:
```
1.
2.
3.
```

Writing it as 1-1-1... has 2 advantages:

1. We know if the mdx renderer breaks.
2. For ease of rearranging steps.